examples/light-switch-app/nrfconnect/README.md

```diff
@@ -0,0 +1,593 @@
+
+    $ chip-tool pairing ble-thread <node_id> hex:<operationalDataset> <device bluetooth pin code> <device bluetooth discriminator>
+
+Fill up given command with your settings:
+- __< node_id >__ is an unique device node id on thread network.
+- __< operationalDataset >__ is a operational data set of thread network to get it 
+- __< device bluetooth pin code >__ - is a pin code set in _example_directory/main/include/CHIPProjectConfig.h_ file and its default value is _20202021_
+- __< device bluetooth discriminator >__ is a unique ble discriminator set in _example_directory/main/include/CHIPProjectConfig.h_ and its default value is _3840_
+
+Example of pairing command using default settings:
+
+    $ chip-tool pairing ble-thread 1 hex:<operationalDataset> 20202021 3840
+
+After commissioning of lighting app and light switch use following steps to bind devices:
+
+_In these steps following variables should be replace_:
+ 
+ - < lighting id > &mdash; node_id of lighting-app used in commissioning
+ - < switch id > &mdash; node_id of light_switch-app used in commissioning
+
+ 
+1) Add ACL (Access Control List) in Lighting-App device:
+
+To give the lighting endpoint permissions to receive commands from light switch an ACL should be written. An ACL should contain information about all clusters which can be called by light-switch.
+
+Command: 
+
+    $ chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [<switch id>], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}]' <lighting id> 0
+
+
+Where:  
+__{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}__ is an ACL for communication with chip-tool 
+
+__{"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [< switch id >], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}__ is an ACL for binding (cluster nr 6 is a _onoff_ cluster and nr 8 is a _LevelControl_ cluster).
+ 
+To check if ACL was added use command:
+
+    $ chip-tool accesscontrol read acl <lightning id>  0 
+
+ 2) Add binding table:
+    
+Command for unicast binding:
+
+    $ chip-tool binding write binding '[{"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 6}, {"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 8}]' <light switch id> 1
+
+Where:  
+__{"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 6}__ is a binding for __onoff__ cluster 
+
+__{"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 8}__ is a binding for __LevelControl__ cluster. 
+
+To use light switch without brightness dimmer apply only first binding command.
+
+ If Light switch device is rebooting a binding table is restoring from flash memory and device tries to bind known lighting-app device.
```

Comment:
All these lines need to be reworked into clear steps, not a series of paragraphs. Check for example the formatting here: https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/nrfconnect_examples_software_update.md